### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by LanaDX**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,23 +1,51 @@
...
+# GridView for Web Forms - How to update an external control during a callback
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/e2379/)**
 <!-- run online end -->
 
+A control can update only its own rendering in a callback. This example shows how to update the [ASPxLabel](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxLabel) control when a callback of the [ASPxGridView](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView) control is finished.
+
+![Label updated on grid callback](updated-label-on-grid-callback.png)
+
+[ASPxGridView](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView) sends callbacks to update row data on the server. The control raises the [EndCallback](https://docs.devexpress.com/AspNet/js-ASPxClientGridView.EndCallback) client event after a callback is successfully executed. You can handle the event to perform custom actions with another control when a callback is finished.
```
Comment (albertov05) [inline Readme.md]:
```suggestion
[ASPxGridView](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView) sends callbacks to update row data on the server. The control raises the [EndCallback](https://docs.devexpress.com/AspNet/js-ASPxClientGridView.EndCallback) client event after a callback is successfully executed. You can handle the event to perform custom actions with another control when the callback is finished.
```

File: Readme.md [hunk 2]
```diff
@@ -1,23 +1,51 @@
...
+```aspx
+<dx:ASPxGridView ...>
+    <ClientSideEvents EndCallback="OnEndCallBack" />
+</dx:ASPxGridView>
+```
 
-<br/>
+The [ASPxGridView](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView) implements the [JSProperties](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridBase.JSProperties) property that allows you to pass information from the server to the client. In this example, an updated row’s key is saved to the [JSProperties](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridBase.JSProperties) property on the server and then this value is used on the client to display information about the updated row.
```
Comment (albertov05) [inline Readme.md]:
```suggestion
[ASPxGridView](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView) implements the [JSProperties](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridBase.JSProperties) property that allows you to pass information from the server to the client. In this example, an updated row’s key is saved to the [JSProperties](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridBase.JSProperties) property on the server. This value is then used on the client to display information about the updated row.
```

File: Readme.md [hunk 3]
```diff
@@ -1,23 +1,51 @@
...
+    if (e.Exception == null) {
+        ((ASPxGridView)sender).JSProperties["cpIsUpdated"] = e.Keys[0];
+    }
+}
+```
+
+```js
+function OnEndCallBack(s, e) {
+    if (s.cpIsUpdated != '') {
+        clientLabel.SetText('The category ' + s.cpIsUpdated + ' is updated successfully');
```
Comment (albertov05) [inline Readme.md]: Can we remove the article ("The") in this message?

File: Readme.md [hunk 4]
```diff
@@ -1,23 +1,51 @@
...
-<!-- default file list end -->
-# ASPxGridView - How to update an external control during a callback
+# GridView for Web Forms - How to update an external control during a callback
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/e2379/)**
 <!-- run online end -->
 
+A control can update only its own rendering in a callback. This example shows how to update the [ASPxLabel](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxLabel) control when a callback of the [ASPxGridView](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView) control is finished.
+
+![Label updated on grid callback](updated-label-on-grid-callback.png)
```
Comment (albertov05) [inline Readme.md:8]: "The category 2 is updated successfully"

Can we remove the article ("The") in this message?
(See my comment below as well about changing the message in your code.)

"Category 2 is updated successfully"

File: Readme.md (hunk 3 above)
Comment (LanaDX) [reply Readme.md]:
```suggestion
        clientLabel.SetText('Category ' + s.cpIsUpdated + ' is updated successfully');
```

File: Readme.md (hunk 4 above)
Comment (LanaDX) [reply Readme.md:8]: Updated in code and image

**Pull request merged**